cmd/snap-seccomp/main.go

```diff
@@ -625,10 +632,16 @@ func parseLine(line string, secFilter *seccomp.ScmpFilter) error {
 		conds = append(conds, scmpCond)
 	}
 
+	// When @squash rules are detected, deny the request
+	act := seccomp.ActAllow
+	if squashRule {
+		act = seccomp.ActErrno
```

Comment:
does this still trigger a message though? that's unclear to me from reading the libseccomp-golang documentation